config/projects/datadog-agent.rb

```diff
@@ -115,6 +144,27 @@
   dependency 'sysstat'
 end
 
+# Ship supervisor anywhere but on Windows
+if not windows?
+  dependency 'kafka-python'
+  dependency 'python-gearman'
+  dependency 'snakebite'
+  dependency 'supervisor'
+
+  # Technically these ones should be shipped on Windows too at some point...
+  # if we ever happen to have a customer use Postgre/pg_boucer on that platform :)
+  dependency 'psycopg2'
+  dependency 'pg8000'
+  dependency 'zlib'
+else
+  # We use our own supervisor shipped as a py2exe-built executable on Windows...
+  # therefore we need py2exe. We also need psutil for our home-made supervisor.
+  dependency 'pywin32'
+  dependency 'py2exe'
+  dependency 'psutil'
```

Comment:
`psutil` is already a dep on non-windows platforms, so I'm not sure we should re-define it here (we could just add a comment on the existing psutil dep declaration below). Up to you though.